content/courses/program-optimization/program-architecture.md

```diff
@@ -385,9 +385,9 @@ structs at the end of the account.
 
 #### For Future Use
 
-In certain cases, consider adding extra, unused bytes to you accounts. These are
-held in reserve for flexibility and backward compatibility. Take the following
-example:
+In certain cases, consider adding extra, unused bytes to your accounts. These
+are held in reserve for flexibility and backward compatibility. Take the
+following example:
```

Comment:
Can't we resize accounts now? We should talk about that. 

Is this advice outdated?

Reply:
Have added details on account resizing with realloc and versioning of accounts for upgrade in future.